docs/fastdds/property_policies/non_consolidated_qos.rst

```diff
@@ -11,11 +11,63 @@ Before consolidating a new QoS Policy, it is usually set using this generic QoS
 Consequently, this section is prone to frequent updates so the user is advised to check latest changes after upgrading
 to a different release version.
 
-``PushMode`` QoS Policy
-^^^^^^^^^^^^^^^^^^^^^^^
+.. _push_mode_qos_policy:
+
+``Push Mode`` QoS Policy
```

Comment:
I'd call the section something like DataWriter operating mode. Also, I think that a link to this section from the datawriter qos section may be appropriate.

Reply:
Done on 21ac05c182abbd86308753aaca6b350f551597c5